resources/sass/app.scss

```diff
@@ -137,6 +137,29 @@ $color-white: #fff;
 	}
 }
 
+.team-info {
+	margin-bottom: 30px;
+	display: flex;
+	flex-wrap: wrap;
+
+	.image-container {
+		width: 18%;
+		margin-right: 2%;
+	}
+
+	.team-people {
+		width: 80%;
+		float: right;
```

Comment:
Si estas usando flex puedes usar algo como `justify-content` en vez de flotar, you might want to take a look at that